packages/core/src/renderer/components/+helm-releases/release-details/release-details-content.tsx

```diff
@@ -35,6 +35,10 @@ interface Dependencies {
 const NonInjectedReleaseDetailsContent = observer(({ model }: Dependencies & ReleaseDetailsContentProps) => {
   const loadingError = model.loadingError.get();
 
+  useEffect(() => {
+    model.load();
+  }, []);
```

Comment:
Do we not call `model.load()` anywhere else? It seems strange that we should have to call it here again.

Reply:
We call it in `releaseDetailsModelInjectable` once https://github.com/lensapp/lens/blob/master/packages/core/src/renderer/components/%2Bhelm-releases/release-details/release-details-model/release-details-model.injectable.tsx#L53

Adding `model.load()` here mean that we fetch release details every time this component rendered == release details opened.

Reply:
A better fix would be to use `asyncComputed` which automatically gets called again when it goes from UNOBSERVED -> OBSERVED. That way this technical details don't get polluted into the component.

Reply:
Moved loading away from model injectable to `getProps` inside `ReleaseDetailsContent`.